### ISSUE TITLE
Page load speed

### ISSUE THREAD

**Issue by rolka** [folkhack/Gravity-Forms-No-CAPTCHA-reCAPTCHA#22]
Hi,

I've got about 30 forms on one page. The problem is that this plugin makes lots of requests and my page freezes for about 20 seconds.

How can I fix this?

Thanks :)

**Comment by joshuaiz** [comment]
With all due respect to the plugin creators, this plugin is no longer necessary as the default CAPTCHA from Google is now NoCAPTCHA. You can safely remove the plugin and just use the built-in CAPTCHA field in Gravity Forms.

**Comment by rolka** [comment]
@joshuaiz  true, you can use built-in CAPTCHA field in Gravity Forms, but it doesn't support Ajax.. that is the problem :)

**Comment by rolka** [comment]
How do I dequeue gf-no-captcha-recaptcha-public.js?

**Comment by rolka** [comment]
ok, you just have to do like
`add_action( 'gform_enqueue_scripts', 'dequeue_no_captcha_recaptcha_public_js', 11 );
function dequeue_no_captcha_recaptcha_public_js() {
    wp_dequeue_script( 'no_captcha_recaptcha_internal' );
}`
